op-program/host/config/config.go

```diff
@@ -248,7 +332,8 @@ func NewConfigFromCLI(log log.Logger, ctx *cli.Context) (*Config, error) {
 		if len(agreedPrestate) == 0 {
 			return nil, ErrInvalidAgreedPrestate
 		}
-		l2OutputRoot = crypto.Keccak256Hash(agreedPrestate)
+		//l2OutputRoot = crypto.Keccak256Hash(agreedPrestate)
+		return nil, errors.New("l2.agreed-prestate is not yet supported")
```

Comment:
Why is this not supported? The hash of the agreed-prestate is the agreed claim which is what l2OutputRoot was used for.  The `l2OutputRoot` variable name is pre-interop but the concept is right and it needs to be provided as the `L2OutputRootLocalIndex` preimage oracle value.

Reply:
It's temporarily not supported. I'd like to introduce new interop-specific flags to configure the other fields in the `InteropInputs` struct. And partially supporting `l2.agreed-prestate` but not the remaining flags is a recipe for misuse.
The end goal is to support `l2.agreed-prestate`.